tests/fv3_conf/cpld_control_run.IN

```diff
@@ -183,8 +183,8 @@ if [ $IAER = 1011 ]; then
 fi
 
 cp    @[INPUTDATA_ROOT]/FV3_input_data/ugwp_c384_tau.nc ./ugwp_limb_tau.nc
-cp    @[INPUTDATA_ROOT]/${FV3_DIR}/INPUT_L127/oro_data_ls* ./INPUT
-cp    @[INPUTDATA_ROOT]/${FV3_DIR}/INPUT_L127/oro_data_ss* ./INPUT
+cp    @[INPUTDATA_ROOT]/${FV3_DIR}/INPUT_L127_gfsv17/oro_data_ls* ./INPUT
```

Comment:
@JongilHan66 @grantfirl The GWD scheme is updated for all the resolution coupled RT tests (c48,/c96/c192/c384) in this PR, is it intended that only C96 oro_data ls/ss files are updated. Would you please confirm if the new version of oro_data ls/ss data for other resolutions is also working in the coupled RT tests? Thank you.

Reply:
> @JongilHan66 @grantfirl The GWD scheme is updated for all the resolution coupled RT tests (c48,/c96/c192/c384) in this PR, is it intended that only C96 oro_data ls/ss files are updated. Would you please confirm if the new version of oro_data ls/ss data for other resolutions is also working in the coupled RT tests? Thank you.

@junwang-noaa the new oro data ls/ss works for all resolutions

Reply:
@JongilHan66 That's great, thanks for confirming. @jkbk2004 since the new oro data works for all the resolutions. Would you please update the oro data ls/ss for all the coupled RT tests so that we can have consistent version oro_ls/ss data? The data directory is at: /scratch1/NCEPDEV/global/glopara/fix/ugwd/20240624 according to @JessicaMeixner-NOAA

Reply:
@junwang-noaa New oro files are available at all machines: e.g. hera: /scratch2/NAGAPE/epic/UFS-WM_RT/NEMSfv3gfs/input-data-20240501/FV3_input_data${ATMRES#C}/INPUT_L127_gfsv17.

Reply:
Great! Please make sure the data is used in the coupled RT tests, let us know if you need help. Thanks